.github/workflows/pack.yml

```diff
@@ -0,0 +1,45 @@
+name: Pack extensions
+
+on:
+  push:
+    paths-ignore:
+      - README.md
+  pull_request:
+    paths-ignore:
+      - README.md
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [21.x]
+    container: archlinux:base-devel
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Cache
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: node_modules
+
+      - name: Prepare
+        run: |
+          pacman -Sy
```

Comment:
You could simplify this into a single line by using `pacman -Sy --noconfirm just`

Reply:
Also consider using this:

https://github.com/marketplace/actions/setup-just

Since this also works on Ubuntu, it would allow to easily get rid of the Arch container